helm/airflow/values.yaml

```diff
@@ -668,56 +673,4 @@ dags:
     existingClaim: ~
   gitSync:
     enabled: false
-    # git repo clone url
```

Comment:
I wouldn't delete things from this values file. In fact, we should have a `cas-airflow` chart that depends on the `airflow` chart, and set the values in `cas-airflow`. If any changes to the `airflow` chart are required, then they should be contributed upstream